storage-api/src/test/java/com/linagora/calendar/storage/ResourceDAOContract.java

```diff
@@ -38,14 +39,14 @@ public interface ResourceDAOContract {
     default void insertShouldAddNewResource() {
         ResourceInsertRequest request = new ResourceInsertRequest(
             List.of(new ResourceAdministrator(new OpenPaaSId("admin1"), "user")),
-            new OpenPaaSId("creator1"),
+            new OpenPaaSId("659387b9d486dc0046aeff21"),
```

Comment:
Please extract the OpenPaSIds into named constants to keep the contract readable (here constant `CREATOR1`, `DOMAIN1`)